modules/RunAlmanack.nf

```diff
@@ -46,7 +49,7 @@ process RunAlmanack {
     echo "Output directory: ${out_dir}" >&2
 
     # Install git and pip
-    apt-get update && apt-get install -y git
+    apt-get update && apt-get install -y --no-install-recommends git && rm -rf /var/lib/apt/lists/*
 
```

Comment:
Is there a hosted container image that has the python version you need and `git`? I think you typically want to avoid installing things in a Nextflow process.